dist/obs-server.spec

```diff
@@ -482,10 +483,9 @@ for i in production.rb ; do
   fi
 done
 
-if [ ! -e %{secret_key_file} ]; then
-  pushd .
-  cd /srv/www/obs/api/config
-  ( umask 0077; RAILS_ENV=production bundle.ruby2.5 exec rails.ruby2.5 secret > %{secret_key_file} ) || exit 1
+if [ ! -s %{secret_key_file} ]; then
+  pushd /srv/www/obs/api
+  RAILS_ENV=production bundle.ruby2.5 exec rails secret > %{secret_key_file}
```

Comment:
Can we use one method? `binstubs` or `bundle exec` calls?

Reply:
yeah - what do you prefer?

Reply:
binstubs

Reply:
5a4bd6a then